addons/test_mail/models/test_mail_models.py

```diff
@@ -37,6 +37,19 @@ class MailTestSimpleWithMainAttachment(models.Model):
     _inherit = ['mail.test.simple', 'mail.thread.main.attachment']
 
 
+class MailTestSimpleUnfollow(models.Model):
+    """ A very simple model only inheriting from mail.thread when only
+    communication history is necessary with unfollow link enabled in
+    notification emails even for non-internal user. """
+    _description = 'Simple Chatter Model'
+    _name = 'mail.test.simple.unfollow'
+    _inherit = ['mail.thread']
+    _partner_unfollow_enabled = True
+
+    name = fields.Char()
+    email_from = fields.Char()
```

Comment:
Maybe we could add a company_id field, just to have a basis when we move on a more MC oriented test suite ? Not really crucial, just pointing it out.